catalog/spatial/csw/spatial-csw-endpoint/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/CswEndpoint.java

```diff
@@ -713,43 +725,54 @@ private int updateRecords(UpdateAction updateAction)
             queryRequest = queryFactory.updateQueryRequestTags(queryRequest,
                     schemaTransformerManager.getTransformerSchemaForId(updateAction.getTypeName()));
 
-            QueryResponse response = framework.query(queryRequest);
+            Map<String, Serializable> recordProperties = updateAction.getRecordProperties();
+            Iterable<List<Result>> resultList = Iterables.partition(ResultIterable.resultIterable(
+                    framework,
+                    queryRequest), 64);
```

Comment:
@lessarderic can a system hold 1000 records per user per cswsource in memory at once?
Though I do agree that since the front end does 250 per source that might make sense to align this with that